index.html

```diff
@@ -45,37 +45,41 @@ <h2>Our Partners</h2>
       </div>
     </section>
         <div class="business-partners-display">
```

Comment:
after line 44  a </div> is needed

Reply:
@erkamguresen i have added a closing **< /div>** in line 43 (earlier i had skipped with out putting one ).
thanks for your comment.